packages/url_launcher/url_launcher/lib/url_launcher.dart

```diff
@@ -93,6 +97,7 @@ Future<bool> launch(
     enableDomStorage: enableDomStorage ?? false,
     universalLinksOnly: universalLinksOnly ?? false,
     headers: headers ?? <String, String>{},
+    windowName: windowName ?? "_blank",
```

Comment:
`"_blank"` only makes sense in the web. Please, do not set `"_blank"` as a default value here, otherwise this will break the logic to auto-determine the target on the web version.